### PR TITLE
Describe main convention better

### PR DESCRIPTION
**Pull request by joneshf**
From a discussion we had the other day.

@cachar Is this better? Can it be improved more?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,20 +50,22 @@ Data (and functions related to that data) shared across multiple pages.
 ### `Page.`
 `Page.Writing.Rate.Main`, `Page.Writing.Rate.Update`, `Page.Writing.Rate.Model.Decoder`
 
-A page on the site, which has its own URL. These are not reusable, and implemented using a combination of types from `Data` and modules from `Nri`.
+A page on the site, which has its own URL. These are not reusable, and implemented using a combination of types from `Data` and modules from `Nri`. Comments for usage instructions aren't required, as code isn't intended to be reusable.
```
Comment (cachar) [inline README.md:53]: 🎉  Just this morning, I was wondering why sometimes I needed doc comments and sometimes I didn't!! 😻

File: README.md [hunk 2]
```diff
@@ -50,20 +50,22 @@ Data (and functions related to that data) shared across multiple pages.
...
+- `Page.Preferences.Main` corresponds to the URL `/preferences`.
+- `Page.Teacher.Courses.Assignments.Main` corresponds to the URL `/teach/courses/:id/assignments`.
+    N.B. The `/:id` is dropped from the module name to avoid too much complexity.
 
 #### Non-examples
-- `Page.Teach.WritingCycles.Rate.Main` corresponds to no URL, so finding it from the URL will be hard.
-- `Page.Mastery.Main` corresponds to no URL, so finding it from the URL will be hard.
-
+- `Page.Mastery.Main` corresponds to no URL. We would expect `/mastery` to exist, but it doesn't. Finding the module from the URL will be hard.
+- `Page.Teach.WritingCycles.Rate.Main` corresponds to no URL. We would expect `/teach/writing_cycles/rate` or `/teach/writing_cycles/:id/rate` to exist, but neither do. Finding the module from the URL will be hard.
```
Comment (cachar) [inline README.md:68]: Could we include an explanation of why we have non-examples in our codebase? I.e., explain this is older code and we've exempted it through the `TRIAGE` list in `lib/tasks/lint.rake`, but don't follow this non-example.

**Review by cachar — COMMENTED** [review]
Love the examples and the breakdown of how the file path is supposed to match an actual URL.

I'd like a little clarity on why we have non-examples at all, but otherwise looks great!

File: README.md (hunk 2 above)
Comment (joneshf) [reply README.md:68]: That information feels like it belongs in our internal documentation. Will update it in https://github.com/NoRedInk/wiki/pull/859

**Comment by joneshf** [comment]
Given that https://github.com/NoRedInk/wiki/pull/859 was merged. I'm going to assume everything has been addressed.

Merging away!

**Pull request merged**